### PR TITLE
jk instead of esc

### PR DESCRIPTION
**Pull request by sdsaati**
Hi, I modified a bit your code, to use jk combination instead of Esc button to go from insert mode to normal mode
btw, Thank you for this amazing addon!

### CODE REVIEW

File: addons/godot_vim/plugin.gd [hunk 1]
```diff
@@ -24,16 +24,23 @@ class Cursor:
...
-				return
-		draw_cursor()
-	
+		pass
+
 	func _input(event):
+		if mode == Mode.INSERT_MODE:
+			var old_time = Time.get_ticks_msec()
+			if Input.is_key_label_pressed(KEY_J):
+				print("j is pressed")
```
Comment (bernardo-bruning) [inline addons/godot_vim/plugin.gd:35]: Please remove this print

File: addons/godot_vim/plugin.gd [hunk 2]
```diff
@@ -24,16 +24,23 @@ class Cursor:
...
+				if Time.get_ticks_msec() - old_time < 500 and Input.is_key_label_pressed(KEY_K):
+					print("k is pressed too")
```
Comment (bernardo-bruning) [inline addons/godot_vim/plugin.gd:37]: Please remove this print

**Review by bernardo-bruning — CHANGES_REQUESTED** [review]
Hi @sdsaati thanks for contribute, after somes PRs your PR get some conflicts, can fix this?

**Comment by sdsaati** [comment]
Hi @bernardo-bruning, thank you for the response, I removed those prints and made a new commit, should I do anything else?

**Comment by bernardo-bruning** [comment]
I see conflicts between my branch and yours, the repository has introduced hard changes in the last few days, and resolving conflicts needs to fix to apply your PR.

@sdsaati

**Comment by sdsaati** [comment]
> I see conflicts between my branch and yours, the repository has introduced hard changes in the last few days, and resolving conflicts needs to fix to apply your PR.
> 
> @sdsaati

Thanks, I will try to solve the conflicts, please give me about 2 or 3 days, then I'll apply the results

**Comment by sdsaati** [comment]
Hi, I think now there is no conflicts, this code adds 'jk' as an alternative to 'esc' in order to going to normal mode

**Review by bernardo-bruning — APPROVED** [review]
Great job! Thanks for contributing. 
I will release version 0.3 on Friday with your contribution.

**Pull request merged**